### DISCUSSION TITLE
Hebrew characters?

### DISCUSSION THREAD

**Discussion by leoncurchack** [excalidraw/excalidraw#8663] [General]
Is anyone interested in creating Hebrew characters for this font?  Let me know what I can do to help.

**Comment by Mrazator** [comment]
We would likely have to find some font similar to Excalifont and add it as a fallback, as we did with recent CJK support. If you are aware of some Hebrew font that would go well with the handdrawn experience, let's discuss it.

**Comment by leoncurchack** [comment]
This is the closest I found, but it is not the best.
https://www.hebrewfont.net/download.php?id=737454


------ Original Message ------
From "Marcel Mraz" ***@***.***>
To "excalidraw/excalidraw" ***@***.***>
Cc "leoncurchack" ***@***.***>; "Author" 
***@***.***>
Date 10/22/2024 1:28:08 PM
Subject Re: [excalidraw/excalidraw] Hebrew characters? (Discussion 
#8663)

**Comment by Mrazator** [comment]
Not bad! Though what I forgot to mention is that it needs to be a font for free commercial usage, ideally with SIL Open Font License. Hence unfortunately we cant use this one.
